source/routing/raptor_api.cpp

```diff
@@ -1021,6 +1021,123 @@ parse_datetimes(RAPTOR& raptor,
     return datetimes;
 }
 
+std::vector<Path>
+_call_raptor(navitia::PbCreator& pb_creator,
```

Comment:
it seems that this function may be static

Reply:
static functions are functions that are only visible to other functions in the same file (more precisely the same translation unit). But we have namespace for this function :smile:

Reply:
:white_check_mark: